tensorflow-notebook/values-okteto.yaml

```diff
@@ -2,5 +2,5 @@ jupyter:
   password: okteto
   resources:
     limits:
-      cpu: 1
-      memory: 2
+      cpu: 1.25
```

Comment:
This would not run with the new free tier. The default value should run in the free tier too

Reply:
I tried with default one but in okteto cloud with this config it creates error.

Reply:
ouch! there is a typo, it should be `memory: 2Gi`, not `memory: 2`.
Could you update that?

Reply:
ok